apis/core/lib/domain/cube-projects/create.ts

```diff
@@ -1,20 +1,22 @@
 import { GraphPointer } from 'clownface'
-import { rdfs, rdf, hydra } from '@tpluscode/rdf-ns-builders'
+import { rdfs, rdf, hydra, dcterms } from '@tpluscode/rdf-ns-builders'
 import { cc } from '@cube-creator/core/namespace'
 import { ResourceStore } from '../../ResourceStore'
-import { cubeProject, csvMapping } from '../identifiers'
+import { cubeProject, csvMapping, creatorUser } from '../identifiers'
```

Comment:
I think the command parameter and factory function should be somewhat reversed. 

```suggestion
import { cubeProject, csvMapping, user } from '../identifiers'
```